app/assets/stylesheets/companion_window.scss

```
@@ -454,3 +454,7 @@
     }
   }
 }
+
+.sul-embed-pdf {
+  min-height: 700px;
+}
```

```suggestion
.sul-embed-pdf {
  min-height: 90vh;
}
```

I tested this and it seems to have more or less the same effect on desktop. On mobile, it prevents the scenario where the container is forced to be too large, and so you can scroll down "out of sight" and the scroll handle vanishes. On both platforms, it got computed to about 468px. Try it and see what you think.

---

Thanks for checking this on mobile.  I made your change and it looks great for me on my laptop (localhost).